From 545a2a561522efc2869066792062694b59b1b39c Mon Sep 17 00:00:00 2001
From: Dominique Leuenberger <dimstar@opensuse.org>
Date: Wed, 2 Aug 2023 11:29:47 +0200
Subject: [PATCH] Fix test suite with file 5.45

[   12s] test/python_magic_test.py:53: in assert_values
[   12s]     self.assertIn(value, expected_value)
[   12s] E   AssertionError: 'PDF document, version 1.2, 2 page(s)' not found in ('PDF document, version 1.2', 'PDF document, version 1.2, 2 pages')
---
 test/libmagic_test.py     | 2 +-
 test/python_magic_test.py | 3 ++-
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/test/libmagic_test.py b/test/libmagic_test.py
index 5719a58..7b4665b 100644
--- a/test/libmagic_test.py
+++ b/test/libmagic_test.py
@@ -15,7 +15,7 @@ class MagicTestCase(unittest.TestCase):
     filename = os.path.join(TESTDATA_DIR, 'test.pdf')
     expected_mime_type = 'application/pdf'
     expected_encoding = 'us-ascii'
-    expected_name = ('PDF document, version 1.2', 'PDF document, version 1.2, 2 pages')
+    expected_name = ('PDF document, version 1.2', 'PDF document, version 1.2, 2 pages', 'PDF document, version 1.2, 2 page(s)')
 
     def assert_result(self, result):
         self.assertEqual(result.mime_type, self.expected_mime_type)
diff --git a/test/python_magic_test.py b/test/python_magic_test.py
index d51587c..410a149 100755
--- a/test/python_magic_test.py
+++ b/test/python_magic_test.py
@@ -108,7 +108,8 @@ def test_descriptions(self):
             self.assert_values(m, {
                 'magic._pyc_': 'python 2.4 byte-compiled',
                 'test.pdf': ('PDF document, version 1.2',
-                             'PDF document, version 1.2, 2 pages'),
+                             'PDF document, version 1.2, 2 pages',
+                             'PDF document, version 1.2, 2 page(s)'),
                 'test.gz':
                     ('gzip compressed data, was "test", from Unix, last '
                      'modified: Sun Jun 29 01:32:52 2008',
